### ISSUE TITLE
All partitions not showing

### ISSUE THREAD

**Issue by 124sami** [theintel/twrpwaltonprimoh8#12]
I wanted to install a GSI custom ROM but the system partition is not showing here. Only Boot and Recovery Partition is showing here.
Plz fix it.

![Screenshot_2022-06-23-21-14-35](https://user-images.githubusercontent.com/96975988/175964478-1ea591de-d337-4686-b7bb-dc7c9a73c50a.png)

**Comment by theintel** [comment]
I once tried to  include the system partition for flash. I abandoned the idea, probably because TWRP prompted some sort of error.

I'll look into it again. I'll let you know.

**Comment by theintel** [comment]
Found why my previous attempt failed. This system.img is not a standard img file and it can't be flashed via TWRP. You have to use `fastboot flash system system.img` on a PC.

Details: https://www.hovatek.com/forum/thread-37072.html

**Comment by 124sami** [comment]
Do you have any custom rom for this phone ?

**Comment by theintel** [comment]
Nope. Maybe you can try this: https://www.getdroidtips.com/lineage-os-17-1-walton-primo-h8-android-10/
WARNING: Its untested and may be full of bugs.

**Comment by 124sami** [comment]
Can you port a custom rom for this phone ?

**Comment by theintel** [comment]
No experience
